.github/workflows/ci.yml

```diff
@@ -47,11 +47,8 @@ jobs:
     needs: lint
     strategy:
       matrix:
-        pyver: ["3.6", "3.7", "3.8", "3.9", "3.10"]
+        pyver: ["3.7", "3.8", "3.9", "3.10"]
         os: [ubuntu, macos, windows]
-        include:
-          - pyver: pypy3
-            os: ubuntu
```

Comment:
Do we actually want to drop this? It's only using 3.6 due to the wrong syntax. We can use `pypy-3.8` instead.

Reply:
Ok, thanks!